src/cortex_cli/auth.py

```diff
@@ -95,14 +95,16 @@ def refresh_request(url: str, realm: str, client_id: str, refresh_token: str) ->
     """Sends refresh request to the authentication server.
 
     Raises:
+        Timeout: no response from auth server within the timeout period
```

Comment:
```suggestion
        TimeoutError: no response from auth server within the timeout period
```
?

Reply:
The actual exception is `requests.exceptions.Timeout`, not the builtin `TimeoutError`.